gallery/src/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import packageJson from '../../package.json';
+
+const plaidVillain = require('./plaid-villain.svg');
+const tartanLogotype = require('./tartan-logotype.svg');
+
+import './PageHeader.scss';
+
+const PageHeader = () => (
+  <header className="nx-gallery-page-header">
+    <div className="nx-gallery-page-header__inner">
+      <h1 className="nx-gallery-page-header__logo">
+        <a className="nx-gallery-page-header__home-link"
+           href="#"
+           aria-label="home">
+          <img className="nx-gallery-page-header__logo-image"
+               src={plaidVillain}
+               alt="A cute but villainous Tartan logo" />
```

Comment:
After re-reading [WHATWG's guidance on `alt`](https://html.spec.whatwg.org/multipage/images.html#alt), I still think this image should have an empty alt attribute, while the other image should have "Sonatype Tartan" as its alt attribute (as it does).  The following excerpts from the HTML standard are especially relevant:

> A corollary to this is that the alt attribute's value should never contain text that could be considered the image's caption, title, or legend

> If the logo is being used next to the name of the entity that it represents, then the logo is supplemental, and its alt attribute must instead be empty.

Reply:
> After re-reading [WHATWG's guidance on `alt`](https://html.spec.whatwg.org/multipage/images.html#alt), I still think this image should have an empty alt attribute, while the other image should have "Sonatype Tartan" as its alt attribute (as it does). The following excerpts from the HTML standard are especially relevant:
> 
> > A corollary to this is that the alt attribute's value should never contain text that could be considered the image's caption, title, or legend
> 
> > If the logo is being used next to the name of the entity that it represents, then the logo is supplemental, and its alt attribute must instead be empty.

Hmm, okay, makes sense to follow the guidance: updated it here: c400966dcd9ada6ecae2cc9d764dcb3363e9fe22